Include/object.h

```diff
@@ -461,8 +461,11 @@ static inline void _Py_INCREF(PyObject *op)
 
 #define Py_INCREF(op) _Py_INCREF(_PyObject_CAST(op))
 
-static inline void _Py_DECREF(const char *filename, int lineno,
-                              PyObject *op)
+static inline void _Py_DECREF(
+#ifdef Py_REF_DEBUG
+    const char *filename, int lineno,
+#endif
+    PyObject *op)
 {
     (void)filename; /* may be unused, shut up -Wunused-parameter */
```

Comment:
These suppressions shouldn't be necessary anymore, but if they are, they should be conditional.

Reply:
The suppression are needed with the current code, since filename & fileno are not used for a release build. It's the purpose of my change: don't pass them if we don't need them ;-)